developer-docs-site/docs/guides/prover-guides/spec-lang.md

```diff
@@ -916,14 +810,10 @@ module addr::M {
 }
 ```
 
-As written, the prover will report that the invariant is violated
-after the call to `publish1` and before the call to `publish2`.
-If either of `publish1` or `publish2` is without the other, the prover
+As written, the Move Prover will report that the invariant is violated after the call to `publish1` and before the call to `publish2`. If either of `publish1` or `publish2` is without the other, the Move Prover
```

Comment:
Is "the invariant is violated" correct here?

Reply:
Yes. We could do "report an violation of the invariant".